Python/prworkflow/README.md

```diff
@@ -0,0 +1,33 @@
+# A command line tool for understanding the PR workflow
+
+# setup and run
+- run install.sh
+```shell
+$ bash install.sh
+```
+
+## usage:
+```shell
+$ python -m prworkflow <"username/repo_name"> -<argment> 
+``` 
+
+## optional arguments:
```

Comment:
Can you add the commands in a tabular markdown form? It will be more readable.

Reply:
I don't get it !😅 " Can you please suggest any example README "